File: eternal_return/src/main/java/org/EternalReturn/ERCharacter/Character/jackie/Character_Jackie.java
package org.EternalReturn.ERCharacter.Character.jackie;

import org.EternalReturn.ERCharacter.ERCharacter;
import org.EternalReturn.ERPlayer.ERPlayer;

public class Character_Jackie extends ERCharacter {

    public boolean isBloodSweep = false;

    public Character_Jackie(ERPlayer erPlayer) {
        super(erPlayer);
        registerMonobehaviour(new Passive());
        registerMonobehaviour(new Active());
        registerMonobehaviour(new Attack());
    }

    @Override
    public String getName() {
        return "Jackie";
    }
}

File: eternal_return/src/main/kotlin/org/eternalreturn/ercharacter/character/jackie/Character_Jackie.kt
package org.eternalreturn.ercharacter.character.jackie

import org.bukkit.entity.Player
import org.eternalreturn.ercharacter.datastructure.CoolTableSeconds
import org.eternalreturn.erplayer.DPlayer
import org.eternalreturn.system.EREngine

class Character_Jackie(erEngine : EREngine, player: Player) : DPlayer(player, erEngine) {
    var isBloodSweep: Boolean = false
    override val activeCoolForEachLevel  = CoolTableSeconds(this::activeLevel, 40, 37, 34, 31, 25)
    override val passiveCoolForEachLevel = CoolTableSeconds(this::activeLevel, 0, 0, 0, 0, 0)

    init {
        this.ActiveCooldownSeconds = 8
        this.PassiveCooldownSeconds = 8
        registerMonobehaviour(Passive())
        registerMonobehaviour(Active())
        registerMonobehaviour(Attack())
    }

    override val name: String
        get() = "Jackie"
}
